flexget/plugins/output/json.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import os
+import json
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils.template import render_from_task, get_template, RenderError
+
+PLUGIN_NAME = 'make_json'
+
+log = logging.getLogger(PLUGIN_NAME)
+
+
+class OutputJson(object):
+    schema = {
+        'type': 'object',
+        'properties': {'template': {'type': 'string'}, 'file': {'type': 'string'}},
+        'required': ['file'],
+        'additionalProperties': False,
+    }
+
+    def on_task_output(self, task, config):
+        # Use the default template if none is specified
+        if not config.get('template'):
+            config['template'] = 'json.template'
+
+        filename = os.path.expanduser(config['template'])
+        output = os.path.expanduser(config['file'])
+        # Output to config directory if absolute path has not been specified
+        if not os.path.isabs(output):
+            output = os.path.join(task.manager.config_base, output)
+
+        json.dump(output)
```

Comment:
We need to pass the filename, as well as the object to encode here, which'll be something like `task.accepted`. We might have to do some work on it first to strip out anything that isn't json serializable.

Reply:
I used much of the code from lines 85+ in the `db.py` as a `serialize` method in `entry.py`. I'm not sure if this is what you had in mind. But this pull-request relies on that method. I can move it here, if needed, though.